File: packages/rook_sdk_apple_health/lib/src/data/mapper/rook_environment_mappers.dart
import 'package:rook_sdk_apple_health/src/data/proto/protos.pbenum.dart';
import 'package:rook_sdk_core/rook_sdk_core.dart';

extension RookEnvironmentMappers on RookEnvironment {
  RookEnvironmentProto toProto() {
    return switch (this) {
      RookEnvironment.sandbox => RookEnvironmentProto.SANDBOX,
      _ => RookEnvironmentProto.PRODUCTION,
    };
  }
}

File: packages/rook_sdk_apple_health/ios/Classes/Mapper/RookEnvironmentMappers.swift
//
//  RookEnvironmentMappers.swift
//  Pods-Runner
//
//  Created by Daniel Nolasco on 13/10/23.
//

import Foundation
import RookSDK

extension RookEnvironmentProto {
 
    func toDomain() -> RookEnvironment {
        if self == RookEnvironmentProto.sandbox {
            return RookEnvironment.sandbox
        } else {
            return RookEnvironment.production
        }
    }
}
